code/modules/ai/presets/xeno_presets.dm

```diff
@@ -70,6 +70,12 @@
 	. = ..()
 	AddComponent(/datum/component/ai_controller, /datum/ai_behavior/xeno/ranged)
 
+/mob/living/carbon/xenomorph/scorpionbomber
```

Comment:
You forgot the /ai. I do not think this declaration is even useful

Reply:
Yeah. You're likely right. Strangely enough it did work during testing, despite this...I'll get around to it, thanks Bravemole!